src/CMakeLists.txt

```diff
@@ -30,9 +30,11 @@
 # Version 2.8.3 introduces CMakeParseArguments.
 cmake_minimum_required(VERSION 2.8.3)
 
-# Respect the PATH environment variable when searching for compilers.
-find_program(CMAKE_C_COMPILER NAMES $ENV{CC} gcc PATHS ENV PATH NO_DEFAULT_PATH)
-find_program(CMAKE_CXX_COMPILER NAMES $ENV{CXX} g++ PATHS ENV PATH NO_DEFAULT_PATH)
+# Select a default compiler when CMake does not respect the PATH environment variable for some generators,
```

Comment:
Could you change the comment to
```# Select a default compiler because CMake does not respect the PATH environment variable for some generators,
# e..g. on the compute servers in Basel.```

Reply:
Sorry, my first comment had a formatting issue, so if you got this by email, it might have only the second line. I updated  it on Github.

Reply:
Ok, done.